dagql/dagui/db.go

```diff
@@ -749,6 +758,15 @@ func (db *DB) Call(dig string) *callpbv1.Call {
 	}
 	callPayload, ok := db.CallPayloads[dig]
 	if !ok {
+		creators, ok := db.CreatorSpans[dig]
```

Comment:
very nit, just if you end up having to do more pushes anyways: this seems like a case where it'd be easier to follow the logic if you inverted to an early return case

```go
if ok {
   // return it
}

// fallback looking for creators logic
```

Reply:
good point! just pushed a refactor